BusinessCore.Tests/alloret.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using CarManagement.Builders;
+using CarManagement.Models;
+using CarManagement.Services;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace BusinessCore.Tests
+{
+    [TestClass]
+    public class alloret
```

Comment:
hi @alloret1993 !
Please, rename this class to a more descriptive class name like ALloretTests. And rename the class filename too :blush: